src/themes/darkFunky.js

```diff
@@ -0,0 +1,6 @@
+export default {
+    name: 'Dark Funky',
+    thumbnailUrl: 'darkFunky.png',
+    type: 'dark',
+    theme: {{"sidebarBg":"#055374","sidebarText":"#ffffff","sidebarUnreadText":"#ffffff","sidebarTextHoverBg":"#302e30","sidebarTextActiveBorder":"#196caf","sidebarTextActiveColor":"#ffffff","sidebarHeaderBg":"#1f1f1f","sidebarHeaderTextColor":"#ffffff","onlineIndicator":"#399fff","awayIndicator":"#c1b966","dndIndicator":"#e81023","mentionBj":"#0177e7","mentionColor":"#ffffff","centerChannelBg":"#242424","centerChannelColor":"#dddddd","newMessageSeparator":"#e6a419","linkColor":"#e0fd36","buttonBg":"#e80a89","buttonColor":"#000000","errorTextColor":"#ea1410","mentionHighlightBg":"#8b35bd","mentionHighlightLink":"#2bb99a","codeTheme":"monokai","mentionBg":"#0177e7"}},
```

Comment:
You have one too many curly braces on your themes, and those are still using double quotes

Running `npm run check` and `npm run fix` should help you get things sorted here

Reply:
Fixed!